### PR TITLE
Revert "Pranksimov Lawset"

### PR DESCRIPTION
**Pull request by Hislyna**
Reverts yogstation13/Yogstation#20108 . Requested by a council member. Pranksimov is an admin nightmare, It should not have been approved as it just allows AIs to plasmaflood because its funny to them. This lawset is way too lenient and it serves no purpose but to make an AI to do whatever it wants under the name of "its funny to me." Funniness is way too subjective for anything like this.

**Changelog**

🆑

tweak: Reverts a bad lawset and all existance of it.

/🆑

### CODE REVIEW

File: code/game/objects/items/cards_ids.dm [hunk 1]
```diff
@@ -233,91 +233,6 @@
 		windoor.obj_flags |= CMAGGED
 		return
 
-	if(istype(target, /obj/item/aiModule/core/full/crewsimov))
```
Comment (adamsong) [inline code/game/objects/items/cards_ids.dm:236]: Why are you deleting all this?

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/game/objects/items/cards_ids.dm (hunk 1 above)
Comment (Hislyna) [reply code/game/objects/items/cards_ids.dm:236]: Because its part of the CMAG code for Pranksimov

Comment (Hislyna) [reply code/game/objects/items/cards_ids.dm:236]: (I also just pressed the revert button, I did nothing else)

**Comment by Yarinoi** [comment]
maybe add an additional law to maximize funniness?
plasmaflood might induce all manner of guffaws and heehaws for you and your silicon buds, but i dont think all those crew being burnt alive think it's that funny

**Comment by MajManatee** [comment]
change it to just say "Unless doing so would be funny to the majority of the crew"

**Comment by azzzertyy** [comment]
I don't really understand the point of this?
Couldn't you just wipe the ai and use freeform instead? You need to access an ai upload board either way, so what does this really change?

**Comment by ToasterBiome** [comment]
Going with https://github.com/yogstation13/Yogstation/pull/20182

**Pull request closed without merging**